runtime/doc/windows.txt

```diff
@@ -999,6 +1000,21 @@ list of buffers. |unlisted-buffer|
 		    +	a modified buffer
 		    x   a buffer with read errors
 
+		[flags] can be a combination of the following characters,
+		which restrict the buffers to be listed:
+			+	modified buffers
+			-	buffers with 'modifiable' off
+			=	readonly buffers
+			a	active buffers
+			u	unloaded buffers (overrides the "!")
```

Comment:
Argh, you're right. It's wrong in the original patch, too. Wanna make a PR for it? :)